easybuild/easyconfigs/b/BWA/BWA-0.7.15-foss-2016b.eb

```diff
@@ -0,0 +1,28 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Cyprus Institute / CaSToRC, Uni.Lu/LCSB, NTUA
+# Authors::   George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+# Modified::  Robert Qiao <robert.qiao@adelaide.edu.au> @ adelaide.edu.au/phoenix
+##
+
+name = 'BWA'
+version = '0.7.15'
+
+homepage = 'http://bio-bwa.sourceforge.net/'
+description = """Burrows-Wheeler Aligner (BWA) is an efficient program that aligns
+ relatively short nucleotide sequences against a long reference sequence such as the human genome."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = [('http://sourceforge.net/projects/bio-bwa/files/', 'download')]
+
+checksums = ['fcf470a46a1dbe2f96a1c5b87c530554', 'md5']
```

Comment:
please use a `SHA256` checksum instead, and drop the `md5`, this doesn't do what you think (it specifies a 2nd checksum `md5`, which is never used, rather than specifying the checksum type of the provided checksum value), see also other remark